tests/__fixtures__/locale/locale.fixtures.ts

```diff
@@ -77,4 +78,7 @@ export const mockModel = {
   countryId: 216,
   cultureCode: 'en-US',
   newsletterSubscriptionOptionDefault: false,
+  defaultCulture: 'en-US',
+  defaultSubfolder: '/en-us',
+  requestSourceCountryCode: 'PT',
```

Comment:
Name the key as 'sourceCountryCode'.

Reply:
the key on "first render" is 'requestSourceCountryCode' after the serverInitialState it becomes 'sourceCountryCode' as you said. Can I leave it like this?

Reply:
sourceCountryCode: 'PT',